scripts/generateAssetData/baseAssets.ts

```diff
@@ -109,7 +109,7 @@ export const thorchain: AssetWithNetworkName = {
   symbol: 'RUNE',
   precision: 8,
   color: '#33FF99',
-  icon: 'https://assets.coincap.io/assets/icons/rune@2x.png',
+  icon: 'https://coin-images.coingecko.com/coins/images/6595/large/Rune200x200.png?1696506946',
```

Comment:
blocking: this is an actual asset used at runtime, and is the wrong icon of ERC-20 RUNE. Can we also use the trust wallet assets URL here?

Reply:
Updated in [5fa1cb5](https://github.com/shapeshift/web/pull/7298/commits/5fa1cb537de3788088524efdd98220955cb62029)